perfscale_regression_ci/scripts/scalability/descheduler/local_storage_and_pvc.sh

```diff
@@ -0,0 +1,90 @@
+#/!/bin/bash
+################################################
+## Auth=prubenda@redhat.com qili@redhat.com
+## Desription: Script for creating objects and validating the descheduler operator 
+## moves pods using local storage to new nodes
+## Polarion test case: OCP-44240
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-44240
+## Cluster config: 3 master (m5.xlarge or equivalent) with 3 workers
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/descheduler-local-storage.yml
+## optional PARAMETERS: number of JOB_ITERATION
+################################################ 
+
+set +o pipefail
+
+source ../../common.sh
+source common_func.sh
+source local_storage_and_pvc_env.sh
+source ../../../utils/run_workload.sh
+source ../../custom_workload_env.sh
+
+node=""
+worker_nodes=$(oc get nodes -l node-role.kubernetes.io/worker= -o name)
+i=0
+last_worker=""
+first_worker=""
+for worker in ${worker_nodes}; do
+  if [[ $i -eq 0 ]]; then
+    first_worker=$worker
+  else
+    oc adm cordon $worker
+    last_worker=$worker
+  fi
+  i=$((i + 1))
+done
+
+export STORAGE_CLASS=$(get_storageclass)
+
+##have to have following profiles set
```

Comment:
@paigerube14 Do we need to set/check if these profiles are being set before starting the workload? I see that we specify that we need to have these profiles, but don't see where they are being set. May be you can add a comment here.

Reply:
There are a bunch of set up steps that are specified in the README in this descheduler folder. Because they are outlined and have a lot of steps I'm not sure if we need validation here.. what do you think?

Reply:
@paigerube14 Ah.. i see, may be in that case, can we add a note in the comment section and point to README for setup.
Also, ideal situation might be to have validation. B'coz if someone new is trying and don't set it up correctly, we can fail gracefully and save time chasing wrong problem. 
I am fine either way you decide.

Reply:
Since I came to pull request directly, I missed seeing README - but prolly no harm in making a note to see README for details on how to setup these profiles?

Reply:
Added 2 functions to validate the proper namespace gets created and the profiles are correct

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/regression-test/180/console